redux/warGameSlice.ts

```diff
@@ -53,12 +59,43 @@ export const warGameSlice: Slice = createSlice({
         state.playerTwoCurrentCard = state.cardListPlayerTwo[0];
       }
     },
-    //Action that handles winning
+    //Action that handles winning and also handles moving cards from other player and from current card they chose
+    ifPlayerWon: (state: WarGameState, action: PayloadAction<number>) => {
```

Comment:
can we rename this action? the other ones all start with a present tense verb like set or start.

how about something like finishRound? or scoreRound?

Reply:
Renamed in newer commit